packages/react-radio/src/components/Radio/Radio.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
 import { useRadio_unstable } from './useRadio';
-import { RadioProps } from './Radio.types';
 import { renderRadio_unstable } from './renderRadio';
 import { useRadioStyles_unstable } from './useRadioStyles';
+import type { RadioProps } from './Radio.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
 
 /**
- * A Radio component presents a set of options where only one option can be selected.
+ * Radio component is a wrapper for a radio button with a label.
```

Comment:
Note: the diff in this PR is misleading. The only changes were renames of Radio => RadioGroup, and RadioItem => Radio. Because the old name of RadioGroup is the same as the new name for RadioItem, they appear as edits rather than renames.

Future work is planned for updates to Radio, but for the purposes of this PR, please only review the rename, and not the pre-existing content of these files. Thanks!